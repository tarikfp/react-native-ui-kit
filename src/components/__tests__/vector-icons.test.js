import * as React from "react";
import renderer from "react-test-renderer";
import Icon from "../vector-icons";

describe("tests icon component", () => {
  it("renders icon with type and name", () => {
    const tree = renderer
      .create(<Icon name="arrow-left" type="MaterialCommunityIcons" />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
