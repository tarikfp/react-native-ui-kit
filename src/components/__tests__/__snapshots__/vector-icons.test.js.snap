// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tests icon component renders icon with type and name 1`] = `
<Text
  allowFontScaling={false}
  selectable={false}
  style={
    Array [
      Object {
        "color": "#757575",
        "fontSize": 20,
      },
      undefined,
      Object {
        "fontFamily": "Material Design Icons",
        "fontStyle": "normal",
        "fontWeight": "normal",
      },
      Object {},
    ]
  }
>
  Û∞Åç
</Text>
`;
